models/common/fhir-path-builder.ts

```diff
@@ -0,0 +1,56 @@
+/*
+* Fhir Path Builder, used to extract values from FHIR.
+* see http://hl7.org/fhirpath/ for the specification
+* and https://github.com/HL7/fhirpath.js#readme
+* for the latest implementation
+*/
+
+export class FhirPathBuilder {
+  bundle(): BundlePathBuilder {
+    return new BundlePathBuilder("Bundle.entry.resource")
+  }
+}
+
+class BundlePathBuilder {
```

Comment:
think its worth having a resourcePathBuilder that each of the 3 resources extend (Bundle, MedicationRequest, Patient)? can share the private variable and constructor methods

Reply:
Done ✔️